core/modules/css/KvInterfaceImplMySql.cc

```diff
@@ -568,9 +568,11 @@ std::string KvInterfaceImplMySql::dumpKV() {
         if (row[1].first == nullptr or *row[1].first == '\0') {
             result += "\\N";
         } else {
+/*
             if (strchr(row[1].first, '\n') != nullptr) {
                 throw CssError("KvInterfaceImplMySql::dumpKV - value contains newline");
             }
+*/
```

Comment:
I think this exception was here because dump is not readable by our tools if any value field contains newline. Do you use dumps for anything? If we need dumps we should use better format or maybe just escape newline characters (`\n`) on output